Robot/robot.py

```diff
@@ -0,0 +1,47 @@
+from pygame import Vector2
+import pygame
+
+
+class BasicRobot():
+    '''The BasicRobot class represents a playable basic robot
+with a position, radius and direction.'''
+    def __init__(self) -> None:
+        self.position = Vector2(500, 500)  # Position in coordinates
```

Comment:
The coordinates should be calculated according to the arena size inside the game state